Makefile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-BUNDLE_VERSION = 4.1.14
+BUNDLE_VERSION = 4.2.0-0.nightly-2019-09-26-192831
```

Comment:
Commit log is missing a lot of context explaining why we are switching to stable 4.1 to a nightly 4.2 now.

Reply:
I guess this is just a regular update, since the new bits also have this bundle, @praveenkumar would be able to give more information here.

Reply:
@cfergeau what more details do we require, we are shifting to 4.1 to 4.2 side and right now only nightly bits are available?

Reply:
Yes, something like this, and also that we'll aim for 4.2 in the final release, and that we only do it now because before that there were no publicly available 4.2 bits.